Maven install failed since shardingsphere-tracing-opentracing module

## Bug Report

### Which version of ShardingSphere did you use?
5.0.0-RC1-SNAPSHOT, commit 89fba4b93b6c7731f238ec331334fb156785e37e

### Which project did you use? ShardingSphere-JDBC or ShardingSphere-Proxy?
Both

### Expected behavior
maven install success, with ~/.m2/repository/org/apache/shardingsphere/ not exists

### Actual behavior
```
[INFO] shardingsphere-proxy-bootstrap ..................... FAILURE [  8.570 s]
[INFO] shardingsphere-tracing ............................. SKIPPED

[ERROR] Failed to execute goal on project shardingsphere-proxy-bootstrap: Could not resolve dependencies for project org.apache.shardingsphere:shardingsphere-proxy-bootstrap:jar:5.0.0-RC1-SNAPSHOT: Could not find artifact org.apache.shardingsphere:shardingsphere-tracing-opentracing:jar:5.0.0-RC1-SNAPSHOT in apache.snapshots (https://repository.apache.org/snapshots) -> [Help 1]
```

### Reason analyze (If you can)
`shardingsphere-tracing-opentracing` module is removed from `shardingsphere-tracing` sub-modules in commit 44adab0 (PR #8608 ), but it is still referenced in `shardingsphere-proxy-bootstrap` module.

### Steps to reproduce the behavior, such as: SQL to execute, sharding rule configuration, when exception occur etc.

1. clean ~/.m2/repository/org/apache/shardingsphere/
2. mvn clean install -Dmaven.javadoc.skip=true -B -Drat.skip=true -Djacoco.skip=true -DskipITs -DskipTests -Prelease

### Example codes for reproduce this issue (such as a github link).

--- a/shardingsphere-observability/shardingsphere-tracing/pom.xml
+++ b/shardingsphere-observability/shardingsphere-tracing/pom.xml
@@ -30,5 +30,6 @@
     <name>${project.artifactId}</name>
     
     <modules>
+        <module>shardingsphere-tracing-opentracing</module>
     </modules>
 </project>

